### DISCUSSION TITLE
"outcomes" in Vectr API when I try to extract the data always comes back as null (Vectr version prior to 9.0)

### DISCUSSION THREAD

**Discussion by ApeironDragon** [SecurityRiskAdvisors/VECTR#288] [Q&A]
**Describe the topic**
I am trying to use Vectr's GraphQL API to extract the data about the campaigns and all of their related test cases. I am using this request:

curl -H "Authorization: VEC1 KeyID:SecretKey"      -H "Content-Type: application/json"      -d '{"operationName":"CampaignsByIds", "variables":{"ids":["b5d711eb-2edd-4f03-9a24-25eb5ba66dc6"]}, "query":"query CampaignsByIds($ids: [String]!) { campaignsByIds(db: \"FS_THREAT_INDEX\", ids: $ids) { nodes { id name testCases { id name mitreId outcomes createTime updateTime status } } } }"}'      -X POST https://vectr.internal:8081/sra-purpletools-rest/graphql -k

everything comes back good as you will see in the screenshot below except for the "outcomes" always comes back as null, any idea how I can fix that please? 
**Screenshots**
![image](https://github.com/user-attachments/assets/51d59ff1-4e11-43dc-9f35-90abdd2f2621)

Thank you!

**Comment by SRAPSpencer** [comment]
I would reference the 9.0 API changes page here, outcomes was deprecated and outcomePath is it's replacement. 

https://docs.vectr.io/graphql/9.0-changelog/
